packages/browser-wallet/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Changed
 
 -   In the display of a `deployModule` transaction, the previously titled module hash is now titled module reference.
+-   Updated web-sdk to fix incorrect estimated cost for `deployModule` transaction.
```

Comment:
This is really a _Fixed_ entry.

Reply:
I _Changed_ to the SDK version :upside_down_face: